Makefile

```diff
@@ -6,7 +6,7 @@ FIND             := find . -type f -not -path './build/*' -not -path './.git/*'
 FORCE_GO_MODULES := GO111MODULE=on
 OPEN_DOCS        ?= "-open"
 PYTHON           := build/ve/bin/python
-VERSION          ?= 1.0.0-beta2
+VERSION           = $(shell cat version)
```

Comment:
++

Reply:
You're good with the removal of the conditional assignment, right?

I think if people want to pretend things are a different version, they can do so by modifying their artifacts. I don't think they should do it, so I don't think the Makefile should support it, though ;-)

Merging now, but open to change back, if there's something I'm not seeing

Reply:
Yes